docs/release-notes/8.0.35-27.md

```diff
@@ -0,0 +1,68 @@
+
+## Release highlights
+
+Percona Server for MySQL implements telemetry that fills in the gaps in our understanding of how you use Percona Server to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. Find more information in the [Telemetry on Percona Server for MySQL](../telemetry.md) document.
+
+This merge fixes the following issue:
+
+* [PS-8849](https://jira.percona.com/browse/PS-8849): The server exited with the `Assertion failure: row0mysql.cc:218:len < 256 * 256 thread`.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.35 and included in Percona Server for MySQL are the following:
+
+* Upgraded the linked OpenSSL library to OpenSSL 3.0.10.
+* Removed the printed query string limit to display the characters for a detected deadlock section of the engine status log.
+* Fixed a procession of single-character tokens by an FTS parser plugin.
+
+### Deprecations
+
+The following items are deprecated in this release and using any of these items may cause a warning:
+
+* The `binlog_transaction_dependency_tracking` server system variable
+
+* The `old` and `new` server system variables
+
+* The `--character-set-client-handshake` server variable
+
+* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command
+
+* The `performance_schema_show_processlist` variable
+
+We recommend migrating from these items as soon as possible. They can be removed in a future release.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.35 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html).
+
+## Bug fixes
+
+* [PS-7806](https://jira.percona.com/browse/PS-7806): Tables with column compression broke asynchronous replication.
+
+* [PS-8671](https://jira.percona.com/browse/PS-8671): An ocassional failure of the `innodb.ddl_kill` test on Jenkins.
+
+* [PS-8741](https://jira.percona.com/browse/PS-8741): MySQL 8.0.33 failed in Debug mode while running `innodb.ddl_crash_alter_partition`.
+
+* [PS-8879](https://jira.percona.com/browse/PS-8879): Received an out of memory error when running `ALTER TABLE ... COLUMN_FORMAT COMPRESSED;`.
+
+* [PS-8965](https://jira.percona.com/browse/PS-8965): In some situations, the `keyring_vault` retrieved records for already deleted keys with empty `data` and `deletion_time` set. Ensured that deleted keys were properly processed.
+
+* [PS-8973](https://jira.percona.com/browse/PS-8973): The execution of `audit_log_filter.status_var_log_size` failed if it was run multiple times.
+
+* [PS-8975](https://jira.percona.com/browse/PS-8975): When userstat was enabled, concurrent execution of queries to tables, including queries to `I_S.TABLE/INDEX_STATISTICS` and `FLUSH PRIVILEGES` commands caused deadlocks between internal locks of the Percona Server. Statements in the existing connections were stalled, and new connections to the server failed (Thanks to Pierre POMES for reporting this issue.)
+
```

Comment:
There are also
https://perconadev.atlassian.net/browse/PS-8927
and
https://perconadev.atlassian.net/browse/PS-9011
in the https://perconadev.atlassian.net/browse/RM-1292 blocker list.
Any reason they are not in this list